pom.xml

```diff
@@ -193,13 +193,13 @@
         <dependency>
             <groupId>org.testcontainers</groupId>
             <artifactId>localstack</artifactId>
-            <version>1.17.3</version>
+            <version>1.17.5</version>
```

Comment:
For å pirke litt, så kunne vi vel trekt ut ein property for testcontainers-versjon. Men ikkje farleg.